package.json

```diff
@@ -51,15 +51,15 @@
       }
     ]
   },
-  "types": "dist/index.d.ts",
+  "types": "package/index.d.ts",
   "scripts": {
     "lint": "eslint ./package ./tests",
     "format": "eslint --fix ./package ./tests ./examples",
     "test": "jest ./package ./tests",
     "test:example:todo-list": "jest ./examples/todo-list",
     "test:examples": "jest ./examples",
     "test:watch": "jest ./package ./tests --watch",
-    "build": "microbundle --jsx React.createElement",
+    "build": "microbundle --jsx React.createElement --no-generateTypes",
```

Comment:
```suggestion
    "build-types": "cp ./package/index.d.ts ./dist/index.d.ts",
    "build": "microbundle --jsx React.createElement --no-generateTypes && yarn build-types",
```

Reply:
in general I have used "shelljs" to do any commands related to file system as it makes it agnostic to the OS.
`
 "copy-types": "node copy-type-definition.js",
` 

`
// copy-type-definition.js
#!/usr/bin/env node

const shell = require('shelljs');
shell.cp('-R', './package/index.d.ts/*',  './dist');
`

Reply:
Thanks @Chetan-unbounce

Reply:
good one, I'll check